src/js/pensions/config/form.js

```diff
@@ -444,14 +447,27 @@ const formConfig = {
                 disabilityStartDate: currentOrPastDateUI('Date disability began')
               }
             },
-            hasVisitedVAMC: {
+            'view:hasVisitedVAMC': {
               'ui:title': 'Have you been treated at a VA medical center for the above disability?',
               'ui:widget': 'yesNo'
+            },
+            vamcTreatmentCenters: {
+              'ui:description': 'Please enter all VA medical centers where you have received treatment',
+              'ui:options': {
+                viewField: MedicalCenterField,
+                itemName: 'Medical Center',
+                expandUnder: 'view:hasVisitedVAMC'
+              },
+              items: {
+                location: {
+                  'ui:title': 'Name and location of VA medical center'
```

Comment:
Should we include the parenthetical about the location here as well (city, state)?